src/version.ts

```diff
@@ -37,38 +37,24 @@ export function setPackageJsonVersion(config: Config = getConfig()): void {
 
 async function fetchLatestVersion(
   config: Config = getConfig()
-): Promise<GraphQlQueryResponseData> {
+) {
   const [owner, repo] = config.repository.split('/');
 
   let authorization;
   if (process.env.GITHUB_TOKEN) {
     authorization = `token ${process.env.GITHUB_TOKEN}`;
   }
 
+  const octokit = new Octokit({
+    auth: authorization
+  })
+
   try {
-    return await graphql(
-      `
-        query getLatestVersion($owner: String!, $repo: String!) {
-          repository(owner: $owner, name: $repo) {
-            packages(first: 2) {
-              nodes {
-                latestVersion {
-                  id
-                  version
-                }
-              }
-            }
-          }
-        }
-      `,
-      {
-        owner,
-        repo,
-        headers: {
-          authorization,
-        },
-      }
-    );
+    return await octokit.request('GET /repos/{owner}/{repo}/releases/latest', {
```

Comment:
👍🏻 Yip, matches [docs](https://docs.github.com/en/rest/releases/releases#get-the-latest-release).